### PR TITLE
Added signin,signup,erreo page

### PR DESCRIPTION
**Pull request by vrinda-mahajan**
Files to check:
src/components/navbar/navbar.jsx
src/contexts/auth-context.jsx
src/pages/auth/profile.jsx
src/pages/auth/signin.jsx
src/pages/auth/signup.jsx
src/pages/error404/error404.jsx
src/reducers/video-reducer.jsx

### CODE REVIEW

**Review by abhisheksahani-nova — APPROVED** [review]
Looks good nothing much to suggest

File: src/components/navbar/navbar.jsx [hunk 1]
```diff
@@ -18,12 +21,26 @@ const Navbar = () => {
             <i className="input-icon fas fa-search"></i>
         </div>
   
-        <div className="flex-c center">
-        <Link to="/" className="m1 badge-container">
+        {/* <div className="flex-c center">
```
Comment (animeshgarai09) [inline src/components/navbar/navbar.jsx]: Remove commented code here

**Review by animeshgarai09 — APPROVED** [review]
Nothing much to suggest. code is clean and good to merge.

**Comment by vrinda-mahajan** [comment]
Thanks, @abhisheksahani-nova, and @animeshgarai09  for reviewing the PR.

**Pull request merged**